config.toml

```diff
@@ -210,6 +210,54 @@ no = 'Sorry to hear that. Please <a href="https://github.com/kubeflow/website/is
 	icon = "fa fa-calendar"
         desc = "Discussion and help from your fellow users"
 
+[[params.examples.demo]]
```

Comment:
I'm wondering why you've moved the content into the `config.toml` file. Overall, I'd prefer to keep the content in the page rather than in `config.toml`. I think it's simpler for contributors to find the content on the page rather than having to open the file corresponding to the page then find the `config.toml` file. Also, I'm concerned that the `config.toml` file will soon become large and difficult to navigate, when we start adding more samples.

Reply:
Hi Sarah, I understand your concern. I, too, like to see contributors focused on creating content. I removed statics values form the documentation file because I don't think our approach is permanent and  with a bit of work and adequately structured documents, we can automize this process entirely. Do you think we can stay with this for a time and work on document stuctures? I would love to help.

Also We use the `config.toml.` file for global static values in the past,  it seemed like the obvious place. :)

Reply:
Hallo @hefedev :) Thanks for the explanation and +1 to automating the rest of the process. I'm happy to have the parameters in the `config.yaml` file given that it's a step towards fully-automated data retrieval from the GitHub repo. 

It'd be great to have your help with the process going forward.

... and now I see in the latest commit that you've decided to put the content back into the page. No worries, either way works. 

I think that if you do decide to put the content in the config file, then you should put that comment (the one explaining that the content is in the config file) in the page itself, rather than in the README.

Reply:
Hello @sarahmaddox! , thank you for your support. I'd love to help this fantastic team-building platform of the future :) It's the least we can do to show appreciation for your efforts.
Indeed I changed the implementation and reverted to the original project structure, and I am quite happy with what I had.
 As discussed, the versioning is not reliable, but I feel that with date label will provide some hint to the user, and we now have a plan.

I will send my comment on  other points  asap,  I try to stick to this [instructions](https://github.com/kubeflow/website#creating-and-updating-the-kubeflow-docs) while working at website PRs. 
Thank you once again for helping me get up to speed.

Reply:
Sounds good!